uproot_methods/convert.py

```diff
@@ -38,6 +41,9 @@ def types(cls, obj):
         elif any(x == ("uproot_methods.classes.TH1", "Methods") or x == ("TH1", "Methods") for x in types(obj.__class__, obj)):
             return (None, None, "uproot.write.objects.TH", "TH")
 
+        elif any(x == ("uproot_methods.classes.TH2", "Methods") or x == ("TH2", "Methods") for x in types(obj.__class__, obj)):
```

Comment:
This block might be unnecessary because every TH2 depends on TH1 and the condition for TH1 above this will probably handle all cases. 
Have you found any such case where the condition above this is not able to correctly handle TH2s?

Reply:
Hmm, I'm not sure that's right? There's no `TH` inheritance in uproot-methods, so these classes are independent. This block yields `(module, class)` for each of the object types, and since the `TH2` object is in the `TH2.py` module the previous block will not trigger.

In any case, this condition just allows for writing when you already have an instance of the `TH2` class (probably quite rare I imagine?), i.e

```python
from uproot_methods.classes.TH2 import from_numpy
data = np.random.normal(0, 1, (1000000, 2))
hist = np.histogram2d(data[:,0], data[:,1], bins=(50, 100), range=((-5, 5), (-5, 5)))
h = from_numpy(hist)

f = uproot.create("file.root")
f['hist']= h  # This triggers the call to uproot_methods.convert.towritable
```

Reply:
My mistake. Thanks for pointing it out. (I was unable to catch this as all the histograms I tested came out of ROOT)